src/supplemental.js

```diff
@@ -53,8 +53,10 @@ function collectSkillInfo(damageArray, {remainHP = 1.0}={}) {
 
 function tableHeader([key, type, additionalVal], locale) {
     var str = intl.translate(key, locale);
-    str += intl.translate(("supplemental_" + type), locale);
-    str = str.replace("{value}", additionalVal == null ? "" : additionalVal);
+    if (type != undefined) {
+        str += intl.translate("supplemental_" + type, locale);
```

Comment:
`intl` is global environment value, it's unnecessary dependency for supplemental module.
can this change pass translated value?

or that function should be defined in result.js

Reply:
I did not look the place-holder replacement.

- supplemental unit test does not need to check translation, that translate's territory.
  - generally, unit test should not have outer dependency like "intl", it's environment value for tests.
    when change the environment that effect to tests.
  - unit test does only logic test. replace place-holder is common task, not supplemental specific
    the supplemental specific part is only the place-holder's key name, or the data record types.
- at this point, tableHeader function can be in result.js  that already has a lot dependencies.
  - this just aim keep supplemental module simple and less dependency.
  - the test for tableHeader would be an integration test. (using same Jest framework is ok)